app/build.gradle

```diff
@@ -33,7 +33,7 @@ android {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
 
-            resValue "string", "BASE_URL", "http://10.0.2.2:44355/api/"
+            resValue "string", "BASE_URL", "http://192.168.0.4:3000/api/"
```

Comment:
Die URLs kommen irgendwann mal alle in eine xml wenn die Api übers Internet ansprechbar wird

Reply:
siehe auch #48